UI/GamepadEmu.cpp

```diff
@@ -583,8 +595,10 @@ void PSPCustomStick::Touch(const TouchInput &input) {
 			usedPointerMask &= ~(1 << input.id);
 			analogPointerMask &= ~(1 << input.id);
 			ProcessTouch(input.x, input.y, false);
+			retval = true;
 		}
 	}
+	return true;
```

Comment:
```suggestion
	return retval;
```

(or else get rid of retval.)

-[Unknown]